trino-s3-proxy/src/main/java/io/trino/s3/proxy/server/signing/ParsedAuthorization.java

```diff
@@ -25,9 +25,9 @@
 import static com.google.common.collect.ImmutableSet.toImmutableSet;
 import static java.util.Objects.requireNonNull;
 
-record ParsedAuthorization(String authorization, String accessKey, String region, String keyPath, Set<String> signedLowercaseHeaders, String signature)
+public record ParsedAuthorization(String authorization, String accessKey, String region, String keyPath, Set<String> signedLowercaseHeaders, String signature)
```

Comment:
In hindsight, `lowercaseSignedHeaders` is a better name for this

Reply:
@vagaerg please do this as well

Reply:
Will do